space.n3

```diff
@@ -0,0 +1,141 @@
+A given application typically stores information in several different
+workspaces, some being user private, some shared, and some public.
+""".
+
+:Workspace owl:disjointUnionOf (:PrivateWorkspace :SharedWorkspace :PublicWorkspace).
+
+:uriPrefix a rdf:Property, owl:DatatypeProperty;
+    s:label "URI prefix";
+    ui:prompt "Give the first part of the URis in this workspace";
+    s:comment """URIs which start with this string are in this workspace or storage.
+This may be used for constructing URIs for new storage resources.
+""".
+
+:MasterWorkspace s:subClassOf :PrivateWorkspace;
+    s:label "Master Workspace"@en;
+    s:comment """This is a workspace for storing the
+    information about the other workspaces.
+    As a user, you normally don't have to worry about it.""".
+
+:masterWorkspace a rdf:Property;
+    s:label "master workspace"@en;
+    s:domain foaf:Agent;
+    s:range :MasterWorkspace.
+
+:workspace a rdf:Property;
+    s:label "workspace"@en;
+#    s:domain foaf:Agent;
+    s:range :Workspace.
+
+:PrivateWorkspace s:subClassOf :Workspace;
+    s:label "Private workspace";
+    s:comment """Access only by the you, the user.""".
+
+:SharedWorkspace s:subClassOf :Workspace;
+    s:label "Shared workspace";
+    s:comment   """Access is to some but not all people.""".
+
+:PublicWorkspace s:subClassOf :Workspace;
+    s:label "Public workspace";
+    s:comment   """Aceess is open to the public. Anything in a public workspace
+    can be accesed by anyone.""".
+
+:PreferencesWorkspace s:subClassOf :Workspace;
+    s:label "Preferences workspace";
+    s:comment   """Aceess may not be open to the public. Contains preferences resources.""".
+
+:storage a rdf:Property; s:label "storage";
+    s:range :Storage;
+    owl:inverse [ s:label "workspace included" ];
+    s:comment """The storage in which this workspace is, or the storage which
+contains this resource, or a storage available to this agent to use.""".
+
+##########################################
+#
+#   Storage is a lump out of which workspaces may be carved
+#
+
+:Storage  a s:Class; s:label "storage";
+    s:comment """A storage is a space of URIs in which you have access to data.
+""".
+
+:PublicStorage  a s:Class; s:subClassOf :Storage; s:label "public storage";
+    s:comment """A public storage is a space of URIs in which you have access to data,
+    and all data is accessible to anyone without control.
+""".
+
+:PersonalStorage  a s:Class; s:subClassOf :Storage; s:label "personal storage";
+    s:comment """A personal storage is a space of URIs in which you and only you have access to data,
+    you cannot give access to anyone else.
+""".
+
+:ControlledStorage  a s:Class; s:subClassOf :Storage; s:label "access controlled storage";
+    s:comment """A  storage is a space of URIs in which you can individually control for each resource
+    who has access to it.
+""".
+
+########################
+#
+#  Registration
+#
+#  Which apps I am using where
+#
+#  TBD
```

Comment:
Is there separate work underway related to app registration aside from the interop [work](https://solid.github.io/data-interoperability-panel/specification/#appreg)? If so we should see if/where efforts can be combined.

Reply:
Out of scope for this PR.